code/game/objects/objs.dm

```diff
@@ -40,6 +40,9 @@
 	///odds of a projectile hitting the object, if throwpass is true and the object is dense
 	var/coverage = 50
 
+	///Internal deployable item
+	var/obj/item/internal_item
```

Comment:
@TiviPlus how about a get internal item proc ?

Reply:
Oh true that is better do this instead

Reply:
Done